README.md

```diff
@@ -0,0 +1,23 @@
+# nva-publication-api
+
+This repository contains the core services of NVA (Nasjonalt vitenarkiv) related to creating and administering publications (registrations, results).
+
+This is an AWS serverless application.
+
+## Service diagram
+
+## Link to descriptions of modules
+
+## Deployment dependencies
+
+This application is deployed in AWS via CodePipeline. Build/deploy pipelines are defined in [nva-infrastructure](https://github.com/BIBSYSDEV/NVA-infrastructure). Some services depend on services from [nva-common-resources](https://github.com/BIBSYSDEV/nva-common-resources) and [nva-identity-service](https://github.com/BIBSYSDEV/nva-identity-service).
```

Comment:
Should we mention other application we depend on here? As publication-channels? maybe nvi?